Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -49,33 +52,21 @@ class PreplannedMapModel: ObservableObject, Identifiable {
     @Published private(set) var status: Status = .notLoaded
     
     /// The first map from the mobile map package.
-    var map: Map? { 
-        get async {
-            if let mobileMapPackage {
-                if mobileMapPackage.loadStatus != .loaded {
-                    do {
-                        try await mobileMapPackage.load()
-                    } catch {
-                        status = .mmpkLoadFailure(error)
```

Comment:
Setting a load failure here is a side-effect of the asynchronous property getter which isn't good. Best to update this property when a mobile map package is set, and make this property published.